app/src/pages/myNotes/NotesContainer.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { ListItem } from "@material-ui/core";
+import DisplayNote from "./DisplayNote";
+import data from "./data";
+import styles from "./styles.module.css";
+
+/**
+ * NotesContainer component
+ * @constructor
+ */
+
+const NotesContainer = () => {
+  const [active, setActive] = useState(false);
+
+  const Notes = ({ name }) => (
```

Comment:
Please, move Notes Component outside of the NotesContainer. It's hard to navigate between them.
Example: NotesConstructor - returns an array of Note Component (one note).
NoteComponent -  returns one note details